cmd/podman/shared/intermediate_novarlink.go

```diff
@@ -0,0 +1,58 @@
+// +build !varlink
+// +build !remoteclient
+
+package shared
+
+// ToString wrapper for build without varlink
+func (c CRStringSlice) ToVarlink() interface{} {
```

Comment:
Wait, how does any of this work? You're declaring and returning a new empty variable?

Reply:
...Oh, novarlink tag... Eeek